pkgs/development/libraries/isl/0.17.1.nix

```
@@ -1,24 +1,11 @@
 { lib, stdenv, fetchurl, gmp }:
```

```suggestion
{ callPackage, lib, stdenv, fetchurl, gmp }:
```

Some of the older versions are missing this callPackage arg

---

whoops, that slipped. Thanks!